### PR TITLE
Adds docker-compose files.

### PR DESCRIPTION
**Pull request by MicahZoltu**
Also adds TS source map support for stack traces.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+version: "3"
+services:
+  parity-dev-node:
+    build: ./parity-dev-node
+    ports:
+      - "47622:8545"
+      - "47621:8180"
+  # get-dev-node:
```
Comment (adrake33) [inline docker-compose.yml]: Minor typo: should be `# geth-dev-node:`.

File: source/libraries/ContractDeployer.ts [hunk 2]
```diff
@@ -42,11 +42,12 @@ export class ContractDeployer {
     }
 
     public async deploy(): Promise<boolean> {
-        this.testAccounts = await generateTestAccounts(this.testAccountSecretKeys);
+        const accounts = await this.ethjsQuery.accounts();
+        this.testAccounts = accounts;
```
Comment (adrake33) [inline source/libraries/ContractDeployer.ts]: Do you think it's worth keeping `generateTestAccounts` for unit tests, or should I remove it?

Comment (adrake33) [reply source/libraries/ContractDeployer.ts]: Also, should I remove the `secretKeys` being passed into the constructor along with `this.testAccountSecretKeys`?

Comment (MicahZoltu) [reply source/libraries/ContractDeployer.ts]: We may end up bringing back `generateTestAccounts` for unit tests, but we can always just pull them out of git history.  Same with the secret keys I think.  So no need to keep them around for now I think.

File: docker-compose.yml (hunk 1 above)
Comment (MicahZoltu) [reply docker-compose.yml]: Fixed.

**Pull request merged**